packages/apollos-api/src/data/willow-tv/data-source.js

```diff
@@ -1,29 +1,29 @@
 import { RESTDataSource } from 'apollo-datasource-rest';
 
-export default class LiveStream extends RESTDataSource {
-  baseURL = 'https://willowcreek.tv/api/';
+export default class Youtube extends RESTDataSource {
+  baseURL = 'https://content.googleapis.com/youtube/v3';
 
-  async getAll() {
-    return this.get('all', null, { ttl: 86400 });
-  }
+  willSendRequest = (request) => {
+    request.params.set('key', process.env.YOUTUBE_API_KEY);
```

Comment:
I've been trying to avoid leaving configuration that happens via `process.env`, and moving it to the Config.yml (but referencing a ENV variable) could we do that here? 

I think it's more self documenting when we do that :)